vhost-device-scmi/src/vhu_scmi.rs

```diff
@@ -132,6 +137,19 @@ impl VuScmiBackend {
         })
     }
 
+    /// This function is used to register all devices which has can be notified into handler.
```

Comment:
"/// Registers all the devices that can provide notifications." ?

Reply:
Well